src/lib/template-result.ts

```diff
@@ -92,7 +92,7 @@ export class TemplateResult {
             marker;
       }
     }
-    html += this.strings[l];
+    html += this.strings[l].toString();
```

Comment:
There's the `s` variable above that will also need a `toString` call.

Reply:
That's not necessary because [the `.lastIndexOf` on next line](https://github.com/Polymer/lit-html/blob/c9cb653b0a005d5e53e899ab4911e9ac208cc23b/src/lib/template-result.ts#L69) will do the type check for us.

Reply:
Ahh, ok.

Reply:
I didn't add the `toString` for `s` to save the precious 11 characters of the call.
But if you think that this makes the code clearer, I'll gladly add it.

Reply:
I know it's more verbose, but I'd consider being explicit here and just checking for undefined. The error otherwise will be confusing.

```ts
if (this.strings[l] === undefined) {
  throw new Error(`Invalid escape sequence in ${strings.raw[i]}`);
}
html += this.strings[l]
```